bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java

```diff
@@ -220,10 +222,12 @@ protected BigtableClient getBigtableClient(
       TransportOptions dataTransportOptions,
       ChannelOptions channelOptions,
       ExecutorService executorService) {
-
     try {
-      return BigtableGrpcClient.createClient(
-          dataTransportOptions, channelOptions, executorService);
+      RetryOptions retryOptions = channelOptions.getUnaryCallRetryOptions();
+      BigtableGrpcClientOptions clientOptions =
```

Comment:
This seems convoluted. Why is retryoptions in both channeloptions and clientoptions?